<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericVariableItem</class>
 <widget class="QWidget" name="GenericVariableItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>44</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="_nameLineEdit">
        <property name="placeholderText">
         <string>Variable name...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>55</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="_typeComboBox">
        <item>
         <property name="text">
          <string>On-Off</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Integer</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Double</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="_removeButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="test.qrc">
          <normaloff>:/images/wrong.png</normaloff>:/images/wrong.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="test.qrc"/>
 </resources>
 <connections/>
</ui>
